app/models/evidence_with.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const bookshelf = require('../lib/bookshelf');
+
+require('./locus');
+require('./gene_term_annotation');
+
+const EvidenceWith = bookshelf.model('EvidenceWith', {
+	tableName: 'evidence_with',
+	subject_id: function() {
+		return this.morphOne('Subject', 'subject_id', ['locus']);
```

Comment:
don't you need `type` field for morph? Also I think it needs to be 1:1 with table name, so `Locus`